File: Prog52aLangs/Prog52a.jl
print("Enter Length: ")
length = readline()
length = parse(Int64, length)
print("Enter Width: ")
width = readline()
width = parse(Int64, width)
area = length * width
perim = 2 * length + 2 * width
println("Area is $area")
println("Perimeter is $perim")
#=
Enter Length: 41
Enter Width: 73
Area is 2993
Perimeter is 228
=#

File: Prog52aLangs/Prog52a.rb
puts 'Enter Length: '
length = gets
puts 'Enter Width: '
width = gets
length = length.to_i
width = width.to_i

area = length * width
perim = 2 * length + 2 * width

puts "Area is #{area}"
puts "Perimeter is #{perim}"

=begin
Enter Length: 
10
Enter Width: 
21
Area is 210
Perimeter is 62
=end